covasim/sim.py

```diff
@@ -417,18 +445,24 @@ def run(self, do_plot=False, verbose=None, **kwargs):
 
         T = sc.tic()
 
+        self.scaling_vec = [1.0]
```

Comment:
What about initializing `self.scaling_vec` to be `self['pop_scale'] * np.ones(self.npts)`?